### PR TITLE
Add documentation on Livebook Space

### PR DESCRIPTION
**Pull request by hugobarauna**
- [x] Discuss if this should use "https://huggingface.co/datasets/huggingface/documentation-images" for serving the images
- [x] Check if embedding a Youtube video works in Hugging Face's documentation (since it doesn't work in the Github preview)

### CODE REVIEW

**Comment by osanseviero** [comment]
> Discuss if this should use "https://huggingface.co/datasets/huggingface/documentation-images" for serving the images

Yes! You can open a PR to add the images in https://huggingface.co/datasets/huggingface/documentation-images/tree/main/hub (you can do this through the UI)

>  Check if embedding a Youtube video works in Hugging Face's documentation (since it doesn't work in the Github preview)

Yes, you can add a video as in https://github.com/huggingface/transformers/blob/main/docs/source/es/training.mdx?plain=1#L27. When we run the workflow, a bot will add a link to a preview in this PR so you can preview the final results

File: docs/hub/spaces-sdks-docker-livebook.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+1. Give your Space a name
+2. Set its visibility to public
+3. Create your Space
+
+![](https://i.imgur.com/s4E9yDD.png)
+
+This will start building your Space using Livebook's Docker image.
+
+The visibility of the Space must be set to public for the Smart Cells feature in Livebook to function properly. However, your Livebook instance will still be protected by Livebook authentication, which we will set up now.
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-livebook.md:25]: Maybe mention what ` Smart Cells ` is.

**Comment by hugobarauna** [comment]
@osanseviero opened a [new PR](https://huggingface.co/datasets/huggingface/documentation-images/discussions/44) adding the images to the other repo.

**Comment by hugobarauna** [comment]
Hi @osanseviero, I followed your suggestions and did a few other changes here and there. Please take a look.

The only thing missing is changing the src URL of the images. I can do that once [the other PR](https://huggingface.co/datasets/huggingface/documentation-images/discussions/44) gets merged.

**Review by osanseviero — APPROVED** [review]
Awesome! Thanks a lot! 🔥 

I left a comment in the images PR to make sure the experience is fast :) otherwise seems ready to be merged

**Comment by hugobarauna** [comment]
@osanseviero Tks. I closed the original PR for the images. I opened a new one: https://huggingface.co/datasets/huggingface/documentation-images/discussions/45

**Comment by osanseviero** [comment]
Merged!

**Comment by hugobarauna** [comment]
@osanseviero Ok, replaced the image's src URLs with the ones from huggingface/documentation-images 👍 

Now I think this PR is ready to be merged. Is there anything else missing?

**Comment by osanseviero** [comment]
Not at all! We can merge whenever we're ready to launch! :fire:

**Pull request merged**